From 2a609b1d280da5c5602bf987ba152662c2fa1445 Mon Sep 17 00:00:00 2001
From: Gerrit Pape <pape@smarden.org>
Date: Wed, 26 May 2010 11:18:53 +0000
Subject: [PATCH] [JOBS] Fix for job control off warning

There seems to be a problem with the new version of dash
with job control off.  I can't tell if it is just a warning or is a
manifest bug.

usr/dash/trap.c: In function `exitshell':
usr/dash/trap.c:376: warning: suggest braces around empty body in an `if' statement

Signed-off-by: Herbert Xu <herbert@gondor.apana.org.au>
(cherry picked from commit dc2bc17c7bbe3188cfb0c695dc472b39bcf0b27a)

Conflicts:

	ChangeLog
---
 src/jobs.h |    2 +-
 1 files changed, 1 insertions(+), 1 deletions(-)

diff --git a/src/jobs.h b/src/jobs.h
index 26e421d..9c095ea 100644
--- a/src/jobs.h
+++ b/src/jobs.h
@@ -105,5 +105,5 @@ int waitforjob(struct job *);
 int stoppedjobs(void);
 
 #if ! JOBS
-#define setjobctl(on)	/* do nothing */
+#define setjobctl(on) ((void)(on))	/* do nothing */
 #endif
-- 
1.6.0.3

